### PR TITLE
Players setup

### PR DESCRIPTION
**Pull request by fvr1**

### CODE REVIEW

File: src/observers/index.js [hunk 1]
```diff
@@ -1,15 +1,37 @@
 import {map} from 'rxjs/operators';
 import {fromEvent} from 'rxjs';
 
+const updatePlayers = (players) => {
```
Comment (otro-felipe) [inline src/observers/index.js]: I would place these observers inside separate files.

File: src/game/index.js [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
+
+  endTurn = () => {
+    const {turnIndex, players, cardCount} = this.state;
+
+    this.state.turnIndex = (turnIndex + 1) % players.length;
+    this.state.cardCount = getNextCard(cardCount);
+
     this.subject.next(this.state);
-  };
+  }
```
Comment (otro-felipe) [inline src/game/index.js:36]: This line should end with a semicolon. Did you run the linter?

Comment (fvr1) [reply src/game/index.js:36]: creo que al final no jejejeje

**Review by otro-felipe — APPROVED** [review]
Beautiful :star:

**Pull request merged**